neofetch

```diff
@@ -2772,6 +2781,14 @@ dynamicprompt() {
     # Get cursor position
     info_height="$(IFS=';' builtin read -srdR -t 1 -d c -p $'\033[6n\033[c' ROW COL; printf "%s" "${ROW#*[}")"
 
+    # On Windows 10 subsystem for Linux, the default terminal does not
+    # produce any integer output, so a workaround is used.
+    if [[ $info_height =~ ^-?[0-9]+$ ]]; then
+        :
+    else
+        info_height="0"
+    fi
```

Comment:
For some reason I really don't like the workaround I made. I guess a more effective way can be made.